utils.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"errors"
+	"net/url"
+	"strconv"
+)
+
+func setPageBounds(limit int, page int, len int) (int, int) {
```

Comment:
It is a good practice to name output parameters if there's more then 1 non-error return types, especially when both are the same type so signature would look like that: `func setPageBounds(limit int, page int, len int) (lower int, upper int)`.

Also - page should start with 1, not 0  😉

Reply:
kskitek in the task desc. asked for the page to start with 0

Reply:
ah, missed it, sorry